classes/blocks/class-actionpagedummy.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * ActionPageDummy block class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Blocks;
+
+/**
+ * Class ActionPageDummy
+ *
+ * @package P4GBKS\Blocks
+ */
+class ActionPageDummy extends Base_Block {
+
+	/**
+	 * Block name.
+	 *
+	 * @const string BLOCK_NAME.
+	 */
+	const BLOCK_NAME = 'action-page-dummy';
+
+	/**
+	 * ActionPageDummy constructor.
+	 */
+	public function __construct() {
+		add_action( 'init', [ $this, 'register_actionpagedummy_block' ] );
+	}
+
+	/**
+	 * Register HubspotForm block.
```

Comment:
We will need to edit this comment !

Reply:
[Done](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/879/commits/25a895d910a365e01626db4e6ea715f10358a739)!